rpcd/playbooks/roles/rpc_maas/defaults/main.yml

```diff
@@ -411,6 +411,7 @@ maas_apt_packages:
 maas_pip_packages:
   - rackspace-monitoring-cli
   - ipaddr
+  - swift
```

Comment:
So swift hosts are going to have swift installed in two places? That strikes me as rather confusing and creating the potential for bugs. I think it would be better to get the maas plugin(s) to use the venv's Python interpreter. Maybe a line in file task would suffice for now to achieve that with a new card to look at the more general case.